### ISSUE TITLE
It looks like the PulseAudio team finally want to accept your patches now, but can't contact you :-/

### ISSUE THREAD

**Issue by karora** [pali/hsphfpd-prototype#13]
See:

https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/227#note_720242

I believe they have unblocked you also.

Thanks for all of your work on this, which many of us have needed during this particularly difficult year.

**Comment by karora** [comment]
Quoting from the link above:
```
Tanu Kaskinen @tanuk · 1 hour ago
Owner
On Friday, we tried to email the following message to @pali, but it did not go through because he has blocked us:

Hi Pali, We realise that we made a big mistake by not notifying you of the temporary ban. Please accept our sincere apologies for this. This is the first time we have had to do this, and we did not realise the notification would not reach you.

We are still keen on merging your patches, and believe that your decision to close the MR and remove the patches is based on a misunderstanding of our plans for the usage of libsbc and libopenaptx, and preserving the quality you have worked hard to bring to Bluetooth audio on Linux. We hope that you will be willing to talk to us and resolve this.

-- Georg, Tanu, and Arun

Even though it was the first time we ever banned somebody, at least one of us should have thought of informing Pali and explaining to him why we thought it was necessary. While we are firm in our stance with regards to abusive behaviour, as a token of good will we are lifting the ban (it may take a day to do this as it is the weekend).
```

**Comment by domenkozar** [comment]
See also [my offer to help mediate](https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/227#note_722631)

**Comment by pali** [comment]
Closing, see my previous reply where I *already* explained that I'm not iterested in pulseaudio anymore.

**Comment by domenkozar** [comment]
@pali is that because you lost hope or that it makes no difference to you if it's merged?

**Comment by pali** [comment]
Both.

For about one year more people (including pulseaudio contributors/developers) are sending me emails and saying that I should stop communicate with pulseaudio maintainers that it does not make sense.

Also I'm still waiting for the official answer from freedesktop.org admins for explanation what happened. But the only contact person which can be find on gitlab's freedesktop.org help page do not want to discuss with me or say something more.

**Comment by domenkozar** [comment]
> For about one year more people (including pulseaudio contributors/developers) are sending me emails and saying that I should stop communicate with pulseaudio maintainers that it does not make sense.

I feel your despair. Talking over an online chat would help try something new, which is more effective when conflicts arise.

We would chat about the goals of each side and go over how that can be achieved with the current state of things.

Worst case we've all wasted an hour, best case we find a path forward.

**Comment by userofryzen** [comment]
@pali now it's the worst momento to leave the MR. I think they will accept and pulseaudio can improve a lot in Bluetooth side for your patches ans contributions and they only have to accept you. I have to say too that in the IRC the person who said to add an option with gstream it was not in pulseaudio team. At least it's what I have read inside. So please. For us, go the last step. @domenkozar can help I think if they understand you both and technical things to go ahead and get it merge. Please don't forgive people to get quality Bluetooth. It's in your hand now. And I understand everything about your feelings

**Comment by pali** [comment]
@userofryzen: See my reply https://github.com/pali/hsphfpd-prototype/issues/13#issuecomment-741693688 and https://github.com/pali/hsphfpd-prototype/issues/13#issuecomment-741704712

**Comment by userofryzen** [comment]
I hope the freedesktop contact can answer you @pali . It's a shit that you ve forgiven your code to pa. Buy we can not say to you nothing is your work and pa maintainers almost deserved this. At least is what appear in gitlab

**Comment by domenkozar** [comment]
See https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/227#note_726012

**Comment by pali** [comment]
@domenkozar: I read that linked comment and of course I do not follow up those posts. How even can I if they blocked access to it, including all email notifications? And I'm silent on gitlab and have not wrote any sentense? Of course, what do you think how can I put there comment if gitlab admins blocked me? I'm really surprised that somebody who blocked me, want to talk with me and now is complaining that I do not respond. Sorry, but this is ridiculous.

@userofryzen: Still nothing more.

**Comment by domenkozar** [comment]
@pali I understand that your block has been lifted? Are you still blocked?

**Comment by userofryzen** [comment]
@pali  it's supposed that they unbanned and unblocked you days ago. Isn't it tru? Are they lying us?. So you should have access to all the things including PA. It's what they says in the bug report. If not please, say it and I will put your comment there. It's incredible if they are lying us....

**Comment by fooishbar** [comment]
It is true, @pali is unblocked and can now log in to GitLab. I (as a GitLab admin) blocked his account due to the abuse, and on Monday I unblocked it on request from the PA maintainers.

**Comment by pali** [comment]
@fooishbar: Last time I contacted you about problems with gitlab login and inability to find any contact to admins, you just replied that I was blocked due to bad faith and against the code of conduct and that you would not enter into any further discussion with me. So I respected it and have not write to you any reply.

Nobody til now answered anything why, what and when it happened. And til now I even did not know who made decision.

If you think even NOT notifying blocked users about what happen (if you have already rejected discussion and have not tried to ask also other side for arguments and description) is fully correct with your code of conduct (or how you call it) for somebody who tried to improve really poor bluetooth audio support, then I have no reason to do anything with freedesktop. Because for me this behavior looks like also against [code of conduct](https://www.freedesktop.org/wiki/CodeOfConduct/), in part "Focusing on what is best for the community", "Showing empathy towards other community members" and also "Other conduct which could reasonably be considered inappropriate in a professional setting" (I do not think that it is professional that maintainers can start forcing me via private chats to publish my own code for their own and continue it via ex-colleagues who know me, so I publish my code, making me an assumption that block finish after I change mind and publish it...).

**Comment by liskin** [comment]
> continue it via ex-colleagues who know me

If you're referring to my attempt to salvage the situation, then I should say that this was my own idea, as I happen to be a pulseaudio and bluetooth user who cares about this stuff improving. I built Ubuntu/Debian packages in my free time (all my time is now free, as it happens) because I wanted to help get your code properly tested and merged. When I saw the dispute, I tried all communication channels available to me, because I really deeply care about this work being merged for the benefit of us all.

Please stop assuming ill intent where there is none. We're all just people trying to do our best with the limited time, resources and abilities.

**Comment by fooishbar** [comment]
> Nobody til now answered anything why, what and when it happened. And til now I even did not know who made decision.

This is just a total lie. I told you in private mail exactly why you were banned (the persistent personal abuse), what the ban was (blocking you from logging into GitLab), and you obviously know when it was.

I told you that, though I would not _personally_ enter into further discussion with you, I had CCed the code of conduct committee who are responsible for these decisions and you were welcome to discuss further with them if you wanted. Which, seemingly, you didn't. (The wiki page you linked to lists the names of all members of the committee too, so that's twice you've seen that.)

The fact you're again spreading misinformation and trying to blame all your problems on everyone else around you, is exactly why I told you that I don't personally want to discuss anything more with you, because I really don't.

Anyway, to repeat what I've said above - after the PulseAudio maintainers reversed their decision and requested you were unblocked, I unblocked you, and you've been unblocked since Monday. So that's another untruth you can stop repeating (or not).

**Comment by userofryzen** [comment]
@fooishbar in defense of Pali I can understand him misunterstanding of the all situation(mail tiwh cc to comitte) and we can not blame him and anyone about this problem. It was some mistaken that can be produced and we, like community(at least I) we can make this like it was not happen at all, because we are humans and we make errors. After all, Pali is the person who wrote almost all the code ( or full I dunno) and PA mantainers are the people who are mantaining the project alive, I am sure that they are not making these errors because yes. So, Pali, please, accept your code in PA since they have apologize with you, and all the community understand both of you I think. 
Outside of this, I am sure that PA mantainers has learnt something new about your problem and they will be able to carry on earlier in the next problem similiar they'll have. Right? @pali 
Regards to all the world. And to the PA maintainer reading, it was an stupid thing I said before about you liying, I don't think so at all. 
Respects to all the world mates, remember why you are here!

**Comment by pali** [comment]
@liskin: Ok, sorry for that.

@fooishbar: I have not got any email where you would notify me that you have unblocked me. So do not say that this is local lie, I really have not get it!

I got only that mail where you wrote that you would not talk with me and CCed committee. But nobody from committee have replied to me back yet, I have been waiting and I wrote it here. Also I have not got any notification email from you that I was blocked prior I sent that one my email about problems with login. So if you have really sent them then I'm saying that I have not received them.

You wrote that you have unblocked me. I tried it and gitlab just print error message "Your account has been blocked. Please contact your GitLab administrator if you think this is an error.", without any button how to contact admin. So either you are lying here or there is a technical problem with infrastructure which disallow me to login and which also cause that emails are not delivered. So please do not say to me again that this is total lie.

I have not written here any misinformation or anything else which you think here. And if you do not want to talk with me, I have already wrote that I respect it. But if you really want, you can ask users, e.g. @domenkozar here offered some help and also @userofryzen. Or let users ask you, whatever. But do not say that this is total lie.

Also I would like to know if it is correct when maintainers are multiple times trying to force me via private channels to get from me permission of my work for which I said them that no, I do not.

**Comment by diego-leal** [comment]
Looks like you got banned a second time.

```As of now, I've removed Pali, Andry because of their continued behavior and multiple violations of Code of Conduct. This is a decision that was not made by the PulseAudio maintainers. I stepped in because their overall behavior here is really not acceptable, regardless of how justified their frustrations with this work are.```

Link to comment: https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/227#note_728006

**Comment by userofryzen** [comment]
I'm on fire. And the community too. It's stupid that someone has banned you again. Wtf guys... I am frustrated too. I think all the community is on war with gitlab free desktop now and it's fair. I cant think I'm your frustration...

**Comment by domenkozar** [comment]
I'd kindly ask people not to use name-calling or just any personal attacks, even though we're frustrated and angry. It won't help resolve the conflict, but make things worse.

**Comment by userofryzen** [comment]
Pali, you have not do the things well at all. I have to say. But they are doing the things worse yet. 
The ban topic is flames. Anybody posts if you are or not and if they lifted you the ban and banned again and why and where. It's stupid. Gitlab is working like a fascist website? Is it normal to kick an ass from a FULL site because of a bad moment? And not to lift the band nevermind with him job? 
It's silly man. The best Bluetooth code development and he didn't want you to say there. ,I consent to use my code. I feel frustrated for me. For you. For the community and because it's appear like a dictator website. Because one person's don't like you you are banned and it's all. It's a shit since there is anybody that explain why you were banned and after banned again. So hello? For me is as frustrating that I am thinking going to windows. Because this shit situation is INSANE. Because you want to help a project and you are being blamed... And your work was overrided before being merged. It's... It's... I have no words seriously. Because there is no words that can define your feels or the community ones. Thanks Pali. And thanks to PA maintainers that are trying to go forward and ... Seriously to the shitty people who is banning over ban because of nothing. **** ***

**Comment by userofryzen** [comment]
don't you want to break thinks right now? Because I am on fire right now. This. Is unfair totally unfair and not understandable

**Comment by John-Gee** [comment]
> what the ban was (blocking you from logging into GitLab)


Are you saying that because we was banned by one org, he got banned from all of GitLab? If so, isn't that a bit excessive?

**Comment by userofryzen** [comment]
Exactly it is. And it's what appears to be so.. wtf xD

**Comment by pali** [comment]
Ok, another week passed and seems that maintainer/admin @fooishbar do not want to explain anything, his attitude here is just opposite of the comments from the first two posts. Also comittee has not replied to my original emails for two weeks. So this is ridiculous, how somebody can write such total ballast, block do not answer/explain anything. Therefore I have no reason to change my decision.

**Comment by fooishbar** [comment]
There is nothing more to explain. I blocked you the first time, then unblocked you on later request. After the abuse continued and degenerated, the CoC committee looked more closely at what happened and later made their own independent decision to block you again. This was detailed in comments in the merge request.

I'm unsubscribing from this thread.

**Comment by pali** [comment]
(reply from https://github.com/pali/hsphfpd-prototype/issues/13#issuecomment-749124321)
> I blocked you ... the abuse continued ... decision to block 

Which and where abuse continued? Please show details and the proof of your arguments. The only reported abuse here is what was described in https://github.com/pali/hsphfpd-prototype/issues/13#issuecomment-743743420 and still has not been answered nor explained.

(reply from https://github.com/pali/hsphfpd-prototype/issues/13#issuecomment-749124321)
> then unblocked you

Now I would behave like you and use your words: "This is total lie".

(reply from https://github.com/pali/hsphfpd-prototype/issues/13#issuecomment-739538510)
> We hope that you will be willing to talk to us and resolve this.

(reply from https://github.com/pali/hsphfpd-prototype/issues/13#issuecomment-749124321)
> There is nothing more to explain ... I'm unsubscribing from this thread.

This is really funny. If this is the way how you imagine that people would talk to you then sorry, I have no more reason to willing to talk to you as you wanted. I think that other people have already seen what you are doing here, that you really do not have any interest in resolving anything and that you are finally happy with it.

**Comment by SpineEyE** [comment]
>  After the abuse continued and degenerated, the CoC committee looked more closely at what happened and later made their own independent decision to block you again. This was detailed in comments in the merge request.

@fooishbar you write [above](https://github.com/pali/hsphfpd-prototype/issues/13#issuecomment-743288032) that you unblocked him "on Monday" which can only mean Monday, December 7. How can you say that the abuse continued when the last messages from Pali are from [December 1](https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/issues/36#note_713231), almost 1 week before that?

All you PA/CoC people's decisions don't make much sense and it's okay to make mistakes in a heated situation. I just hope that after 1 or 2 weeks you can look back and recognize that it was more malicious what you did than the angry, venting behavior by pali.

**Comment by userofryzen** [comment]
> There is nothing more to explain. I blocked you the first time, then unblocked you on later request. After the abuse continued and degenerated, the CoC committee looked more closely at what happened and later made their own independent decision to block you again. This was detailed in comments in the merge request.
> 
> I'm unsubscribing from this thread.

I am not agree with you, sorry. maybe in private mails, you have the reason, but here either you either the comitte...

**Comment by John-Gee** [comment]
The PA team now has a new MR supposedly based on yours: https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/440

Does that mean you came to an agreement in the end?

**Comment by pali** [comment]
NO! ABOVE PULL REQUEST IS CONTINUOUS VIOLATION OF POLICES BY PULSEAUDIO MAINTAINERS AND ADMINS. I HAVE NOT ALLOWED THEM TO USE MY CODE FOR THAT ITS BASTARDIZATION AND VIOLATION WHICH THEY ARE USING.

As said more times in this issue I nobody contacted me neither from maintainers admins or committe and tried to explain thins.

They are just stealing my work, blocking me so I would be able to put any reaction and continue this violation.

As you can see in this issue, @fooishbar is there another bastard like other maintainers who is supporting stealing work done by other people and then showing that it is was done by him. And then block and ban people forvever who were done real work to mute them.

I cannot believe what admins, maintainers and committe is doing in "freedesktop".

I respected that they blocked me, but if they continue to do this volation against me, my code and my arguments, I'm going to register another freedesktop account and would continue figthing with them. I will not accept those BASTARDS and their behavior against other people.

Now I'm really angry and I was right if I named them BASTARTS.

**Comment by fooishbar** [comment]
Please stop tagging me in this issue, especially if it’s only to abuse me.

**Comment by pali** [comment]
@fooishbar: Please answer, who give you a permission to steal my code?

I'm warning you: You are doing abusement here. So please immediately explain you actions.

**Comment by fooishbar** [comment]
I haven’t done anything with your code. The first I heard of the new effort
was the notification on the original MR.

If you want to make a moral argument for not using your code as a base,
you’ll need to speak to the author and maintainers. If you want to make a
legal argument, there is none: the license grants the right to use,
redistribute, and modify, as anyone wants. These are the same rights which
allowed you to develop your changes in the first place.

So, again, I don’t know why you’re trying to get me involved, and I have no
idea why you think I’d want to spend my free time at Christmas trying to
help someone who does nothing but call me a bastard. But on that note,
merry Christmas - I hope you find some time to reflect on things and the
way you treat people in particular.

**Comment by pali** [comment]
You have not explained anything nor what other people asked here. You wrote that you were part of people responsible for banning. By your actions you disallowed me to write any reaction to maintainers or speak with them. You were writing there false infromation even after you have been warned. Therefore you are responsible for this situation too.

**Comment by fooishbar** [comment]
Nothing I have written has been incorrect. I'm unsubscribing again. Do not tag me back in.

**Comment by pali** [comment]
I'm warning you again, stop lying. This is not your place where you can ban people which you personally hate and personally would like to steal.

**Comment by fooishbar** [comment]
I haven't lied. I blocked you when I told you I did. I unblocked you when I told you I had as well. Several days after I unblocked you, someone from the fd.o CoC committee looked at what had happened, decided it was unacceptable, and blocked you again.

After that, you created an alternate ban-evading account in order to call me a `BASTARD` and a `bastard` and a `bastard` again.

I do not need to waste any more time on you.

**Comment by pali** [comment]
Stop lying here, this is not your room. You had a chance to explan things. You have did nothing. You have not informed me. You have not unblocked me. I think that everybody can see that your posts are now only total lie. You have violated rules. You are supporting stealing work done by other people. And if somebody show what you are doing, you are only giving bans. I can only name such person as bastard.

**Comment by pali** [comment]
Because @fooishbar is violating rules here, did not want to explain thing and support stealing work done by other people, I'm closing this issue.
